public/js/applicationList.js

```diff
@@ -0,0 +1,86 @@
+const errorMsg = document.getElementById('error-message');
+const proceedBtn = document.getElementById('procced-button');
+const navButton = document.getElementById('applicationsNav');
+const addButton = document.getElementById('addApplication');
+const tableContent = document.getElementById('table-content');
+
+let applicationId;
+
+const openEditApplication = (id) => {
+  // eslint-disable-next-line no-underscore-dangle
+  window.location.href = `${window.location.origin}/views/applicationForm.html?_id=${id}`;
```

Comment:
it is not necessary to use a `_` in the query param name. It is better If you can avoid a lint disable command